scripts/cms-any/license.php

```diff
@@ -12,15 +12,15 @@
 modification, are permitted provided that the following conditions are met:
 
 * Redistributions of source code must retain the above copyright notice, this
-list of conditions and the following disclaimer.
+  list of conditions and the following disclaimer.
```

Comment:
It makes no difference to how the bullet points are rendered if newlines are indented or not, it just looks better in plain text view